### ISSUE TITLE
Unable to run pact on OSX 

### ISSUE THREAD

**Issue by Igor-Pchelko** [pact-foundation/pact-ruby-standalone#54]
# Pre issue-raising checklist

I have already:

* [x] Upgraded to the latest version of the relevant libraries
* [x] Checked to see if the issue has already been raised

## Software versions

* [https://github.com/pact-foundation/pact-ruby-standalone/releases/tag/v1.85.0](https://github.com/pact-foundation/pact-ruby-standalone/releases/tag/v1.85.0)
* Mac OSX 10.15.5

## Expected behaviour

```sh
curl -LO https://github.com/pact-foundation/pact-ruby-standalone/releases/download/v1.85.0/pact-1.85.0-osx.tar.gz
tar xzf pact-1.85.0-osx.tar.gz
cd pact/bin
./pact-mock-service --help start
```

```./pact-mock-service --help start``` Should return help information.

## Actual behaviour

`./pact-mock-service --help start` returns error. 
```sh
./pact-mock-service --help start

/Users/__removed__/pact/pact/lib/ruby/bin/ruby: line 14: D/pact/pact/lib/ruby/lib/ruby/gems/2.2.0: No such file or directory
```

**Comment by Igor-Pchelko** [comment]
Looks like the issue is similar to the: https://github.com/pact-foundation/pact-ruby-standalone/issues/37

But in my case the base directory was `__R&D`. 
And self contained pact can't resolve the path on OSX.

**Comment by Igor-Pchelko** [comment]
May be it's related to the issue with path on OSX: https://github.com/phusion/traveling-ruby/pull/94 ?

**Comment by bethesque** [comment]
Yes, the bug looks correct. I don't know how to get that change into our codebase however. We might have to do some dirty hacks. We're in the process of moving off the Travelling Ruby package however, and replacing it with a Rust shared library. Would you be able to use a different directory in the meantime?

**Comment by Igor-Pchelko** [comment]
Yep, sure.

**Comment by TimothyJones** [comment]
@bethesque - would this diff be the workaround we'd need?

https://github.com/expo/expo-cli/commit/920bb1a68abfca57f553693ea06b54b045d95ea6
